core/domain/exp_services.py

```diff
@@ -1270,3 +1273,44 @@ def search_explorations(query, limit, sort=None, cursor=None):
     """
     return search_services.search(
         query, SEARCH_INDEX_EXPLORATIONS, cursor, limit, sort, ids_only=True)
+
+         
+def _is_suggestion_valid(thread_id, exploration_id): 
+    """Check if the suggestion is still valid. A suggestion is considered 
+    invalid if the exploration version has updated since the suggestion was 
+    made.""" 
+ 
+    states = exp_models.ExplorationModel.get(id=exploration_id).states 
+    suggestion = (feedback_models.SuggestionModel 
+        .get_by_exploration_and_thread_id(exploration_id, thread_id)) 
+    if (suggestion.state_name in states.keys() and 
+           suggestion.state_content == states.get(suggestion.state_name)): 
+        return True 
+    return False        
+
+
+def accept_suggestion(editor_id, change_list, thread_id, exploration_id, 
+                      message):
```

Comment:
s/message/commit_message

and in the body of the method: s/commit_message/full_commit_message